client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts

```diff
@@ -81,9 +84,13 @@ export const useUrlQueryParams = ({
     [updateQuery]
   );
 
-  const updatePaginationQuery = (page: number) => {
+  const updatePaginationQuery = (page: number, pageSize?: number) => {
```

Comment:
Do you think we should keep the save of rows per page? Could persist to local storage here..?

Reply:
Yeah, okay 👍

e56b4d8a7b